internal/pkg/routes/route.go

```diff
@@ -60,10 +60,12 @@ func (r route) Process(data []byte) ([]byte, error) {
 }
 
 func (r route) Consume(msg *nats.Msg) {
-	payload, err := r.Process(msg.Data)
-	if err != nil {
-		r.logger.Error(fmt.Sprintf("Failed to consume msg %s", err.Error()))
-	}
-	topic := fmt.Sprintf("%s/%s", r.MqttTopic(), strings.ReplaceAll(msg.Subject, ".", "/"))
-	r.pub.Publish(msg.Subject, topic, payload)
+	go func() {
+		payload, err := r.Process(msg.Data)
+		if err != nil {
+			r.logger.Error(fmt.Sprintf("Failed to consume msg %s", err.Error()))
```

Comment:
You can pass `err` instead of `err.Error()` here. Also maybe use `consume message`